apps/wing-console/console/app/tests/ex.redis/index.test.ts

```diff
@@ -1,10 +1,25 @@
+import { execSync } from "node:child_process";
+
 import { expect, test } from "@playwright/test";
 
 import { describe } from "../describe.js";
 import { getResourceNode } from "../helpers.js";
 
+const isDockerAvailable = () => {
+  try {
+    execSync("docker ps");
+    return true;
+  } catch {
+    return false;
+  }
+};
+
 describe(`${__dirname}/index.w`, () => {
-  test.skip("opens redis help", async ({ page }) => {
+  test("opens redis help", async ({ page }) => {
+    if (!isDockerAvailable()) {
+      test.skip();
+      return;
+    }
```

Comment:
Does it work adding it to a `beforeAll`?

Reply:
yep 👍

Reply:
yep!